src/main/java/org/jenkinsci/plugins/customizebuildnow/AlternateBuildLabelConfiguration.java

```diff
@@ -0,0 +1,52 @@
+package org.jenkinsci.plugins.customizebuildnow;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+import hudson.Extension;
+import jenkins.appearance.AppearanceCategory;
+import jenkins.model.GlobalConfiguration;
+import jenkins.model.GlobalConfigurationCategory;
+
+@org.jenkinsci.Symbol("customBuildNow")
```

Comment:
import? the place you copied from must have had a ui symbol in the same class

Reply:
fixed